Dockerfile

```
@@ -15,20 +15,31 @@ WORKDIR /usr/src/java-code/opbeans
 #Bring the latest frontend code
 COPY --from=opbeans/opbeans-frontend:latest /app/build src/main/resources/public
 
-RUN mvn -q -B package -DskipTests
+RUN mvn -q --batch-mode package \
+  -DskipTests \
+  -Dmaven.repo.local=.m2 \
+  -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn \
```

We now use 3.6.3 so we can simplify it with

```suggestion
  --no-transfer-progress \
```

See https://maven.apache.org/docs/3.6.1/release-notes.html#user-visible-changes